src/sentry/static/sentry/app/views/performance/transactionVitals/constants.tsx

```diff
@@ -75,7 +75,7 @@ export const WEB_VITAL_DETAILS: Record<WebVital, Vital> = {
       'Captures the time spent making the request and receiving the first byte of the response.'
     ),
     failureThreshold: 600,
```

Comment:
Does requestTime and TTFB actually need a threshold? they don't factor in with web vitals pass/fail do they?

Reply:
I can remove them, though TTFB does have a threshold defined at https://web.dev/time-to-first-byte/